src/tunnel.c

```diff
@@ -1176,9 +1176,8 @@ int ssl_connect(struct tunnel *tunnel)
 #endif
 
 		if (tunnel->config->user_cert) {
-			if (!SSL_CTX_use_certificate_file(
-			            tunnel->ssl_context, tunnel->config->user_cert,
-			            SSL_FILETYPE_PEM)) {
+			if (!SSL_CTX_use_certificate_chain_file(
+			            tunnel->ssl_context, tunnel->config->user_cert)) {
 				log_error("SSL_CTX_use_certificate_file: %s\n",
```

Comment:
Please modify the error message athe same way.

Reply:
Commited as req